unlock-app/src/hooks/useProvider.ts

```diff
@@ -81,6 +82,11 @@ export const useProvider = (config: any) => {
 
   const getWalletService = async (networkId?: number) => {
     const currentNetworkId = Number(network)
+    // If the user is not connected, we open the connect modal
+    if (!isConnected) {
```

Comment:
Do we have a way to "suggest" what the user used in the past?

For example if they connected previously with an unlock account, let's show this one only by default? Etc